app/models/facility.rb

```diff
@@ -200,6 +200,24 @@ def discardable?
     registered_patients.none? && blood_pressures.none? && blood_sugars.none? && appointments.none?
   end
 
+  # For regions compatibility
+  def facility_region?
+    true
+  end
+
+  # For regions compatibility
+  def district_region?
+    false
+  end
+
+  def self.state_region_name(facility_attributes)
```

Comment:
What's this for? Is it used?

Reply:
nope, dead code - removed.

Reply:
Phew, awesome. It was just gnarly enough for me to be thankful that it's dead code 😂